hadoop-ozone/dist/src/main/license/bin/LICENSE.txt

```diff
@@ -355,6 +356,7 @@ Apache License 2.0
    io.prometheus:simpleclient_common
    io.prometheus:simpleclient_dropwizard
    joda-time:joda-time
+   jakarta.inject:jakarta.inject
```

Comment:
Where do we have this dependency?  I only see `org.glassfish.hk2.external:jakarta.inject` in `hadoop-ozone/recon/pom.xml`.

Reply:
Thanks. Removed.